installation_wizard_datachoice

```diff
@@ -0,0 +1,87 @@
+When installing ownCloud Server & ownCloud Enterprise editions the administrator
+may choose one of 3 supported database products.
+
+SQLite
+^^^^^^
+Is the default database for ownCloud Server, but is not available and not supported
+for the ownCloud Enterprise edition.
+
+SQLite will be installed by the ownCloud packages and all the necessary dependencies
+will be satisfied.  See Manual Installation on Linux for a detailed listing of
+required and optional PHP modules.
+
+If you used the packages to install ownCloud, you may "Finish Setup" with no
+additional steps to configure ownCloud using the SQLite database for limited use.
+
+Please note that SQLite is good only for testing and lightweight single user setups.
```

Comment:
not sure if it makes sense, but could use the ".. warning:: " tag here to highlight this